src/ui/qgsvectorlayerpropertiesbase.ui

```diff
@@ -290,7 +296,7 @@
            <string>Legend</string>
           </property>
           <property name="toolTip">
-           <string>Legend</string>
+           <string>Control rendering in legends</string>
```

Comment:
Mm - it's not just rendering though, it also allow creation of new legend items and such. Maybe "Manage the layer's legend"